doc/docportal/use_scummvm/taskbar_integration.rst

```diff
@@ -0,0 +1,47 @@
+===========================
+Taskbar integration
+===========================
+
+The taskbar integration has several features:
+
+- Adds an overlay icon in the taskbar/dock when running a game (Windows and macOS)
+- Updates a list of recently played games that can be started from the taskbar/dock (Windows and macOS).
+- Shows a progress bar when doing a mass add of games, and shows the number of games found at the end (Windows, macOS and Linux).
+- Shows an error state in the taskbar if an error occurs when running a game (Windows only).
+
+.. _iconspath: 
+
+Setting up the icon overlay
+================================
+
+Icons for many games can be found on our `Github repository <https://github.com/scummvm/scummvm-icons>`__. 
+
+Place the icons in a folder, and either change the *iconspath* in the :doc:`configuration file <../advanced_topics/configuration_file>` to point to this folder, or add the folder to the :ref:`Extra Path <extra>` on the Paths settings tab.
+
+The icon files must be ICO files for Windows, or PNG files for macOS. They must follow one of these naming conventions, where xxx is the file extension:
+
+        - :ref:`ID <id>`.xxx
+		- gameid.xxx
+		- engineid-gameid.xxx
+
```

Comment:
I have just merged a pull request that adds the possibility to use `engineid.xxx`

Reply:
Perfect, I'll add that now!